bundle/compliance/policy/aws_iam/validate_credentials.rego

```diff
@@ -0,0 +1,28 @@
+package compliance.policy.aws_iam.validate_credentials
+
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.data_adapter
+import future.keywords.every
+
+duration = sprintf("%dh", [45 * 24]) # 45 days converted to hours
+
+date_format = "2006-01-02 15:04:05.999999999 -0700 MST"
+
+default validate_credentials = false
+
+validate_credentials {
+	assert.array_is_empty(data_adapter.active_access_keys)
+	are_credentials_valid([data_adapter.iam_user])
+}
+
+validate_credentials {
+	are_credentials_valid(data_adapter.active_access_keys)
+	are_credentials_valid([data_adapter.iam_user])
+}
```

Comment:
> For each user having `password_enabled` set to `TRUE` , ensure `password_last_used_date` is less than `45` days ago.

Is this part checked implicitly?

Reply:
> When `password_enabled` is set to `TRUE` and `password_last_used` is set to `No_Information` , ensure `password_last_changed` is less than 45 days ago.

same here. not very clear how we address the audit from the PDF

Reply:
> For each user having an `access_key_1_active` or `access_key_2_active` to `TRUE` , ensure the corresponding `access_key_n_last_used_date` is less than `45` days ago.

same here.

Reply:
I've updated the rule's implementation.
Do you think there's a better way to implement it?